Server/Utilities/CBXDevice.m

```diff
@@ -252,6 +252,11 @@ - (NSDictionary *)formFactorMap {
       @"iPhone10,3" : @"iphone 10",
       @"iPhone10,6" : @"iphone 10",
 
+      // iPhone XS/XS Max/XR - derived from Simulator
```

Comment:
> // iPhone 8/8+/X - derived from Simulator

@ line 247, let's remove the "derived from Simulator"  - that comment is no longer correct. Some of those model numbers are _not_ from the simulator.

Reply:
Fixed.